packages/react/src/patterns/blocks/ContentGroupSimple/README.md

```diff
@@ -29,12 +29,12 @@ const heading = 'Lorem ipsum dolor sit amet';
 
 const mediaData = {
   images: [
-    { src: 'https://picsum.photos/id/2/320/160', minWidth: 320 },
-    { src: 'https://picsum.photos/id/2/400/400', minWidth: 400 },
-    { src: 'https://picsum.photos/id/2/672/672', minWidth: 672 },
+    { src: 'https://dummyimage.com/320x160', minWidth: 'sm' },
+    { src: 'https://dummyimage.com/400x400', minWidth: 'md' },
+    { src: 'https://dummyimage.com/672x672', minWidth: 'lg' },
   ],
+  defaultImage: 'https://dummyimage.com/672x672',
```

Comment:
Styling and ratio?

Reply:
I didn't bother to put that in this part of the README, as I'd have to document the service's API. If you think it should be in the docs we can add a few lines for usage guidance.